.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
```

Comment:
Kindly, set up your linters correctly as mentioned by the previous reviewer according to the [documentation](https://github.com/microverseinc/linters-config/blob/master/html-css/README.md). You have 2 files missing` .hintrc`  and `.stylelintrc.json`